tests/jobs/json/hdfs-kerberos-delete-terasort-files.json

```diff
@@ -1,16 +1,12 @@
 {
   "id": "hdfs-kerberos-delete-terasort-files",
   "description": "Job that deletes Terasort files from Kerberized HDFS",
-  "labels": {},
   "run": {
     "cpus": 0.1,
     "mem": 512,
     "disk": 0,
-    "cmd": "/bin/bash && cd $MESOS_SANDBOX && python3 dcos-cli-auth.py --username=$DCOS_UID --password=$DCOS_PASSWORD && dcos task exec hdfsclient kinit -k -t hdfs.keytab hdfs/name-0-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL && dcos task exec hdfsclient bin/hdfs dfs -rm -r -f /terasort_in /terasort_out",
+    "cmd": "/bin/bash && cd $MESOS_SANDBOX && python3 dcos-cli-auth.py --username=$DCOS_UID --password=$DCOS_PASSWORD && dcos task exec hdfsclient kinit -k -t hdfs.keytab hdfs/name-0-node.data-serviceshdfs-kerberos.autoip.dcos.thisdcos.directory@LOCAL && dcos task exec hdfsclient bin/hdfs dfs -rm -r -f /terasort_in /terasort_out",
```

Comment:
is there a way to template this? seems kinda fragile.

Reply:
That's a good suggestion. I'll template it.